go/src/koding/klient/machine/mount/index/index_std.go

```diff
@@ -0,0 +1,14 @@
+// +build !darwin,!linux
```

Comment:
ok i missed index_darwin file, what keeps us from implementing for others or using a package for that ( i remember there was a package just for unifying them )